actions/bumper/entrypoint.sh

```diff
@@ -185,16 +185,27 @@ else
         version_new=${new}-${version_suffix}
     fi
 
-    version_line=$(cat $version_file_path | grep -e "${version_line_match}")
-    if [ -z "$version_line" ]; then
-        echo "No version line found; no bump of version file."
-    else
-        new_line=$(echo "$version_line" | sed -E -e "s/$version_pattern/\1${version_new}\3/")
-        sed -E -i.bak -e "s/${version_line}/${new_line}/" $version_file_path
-    fi
     git config --global user.email "robot@terra.team"
     git config --global user.name "bumptagbot"
-    git add $version_file_path
+
+    # check for multiple version file paths to update
+    IFS=',' read -ra file_path <<< "$version_file_path"
+    IFS=',' read -ra line_match <<< "$version_line_match"
+    for i in "${!file_path[@]}"; do
+        version_file_path_i=${file_path[$i]}
+        version_line_match_i=${line_match[$i]}
+        echo "Updating $version_file_path_i located at $version_line_match_i"
+        
+        version_line=$(cat $version_file_path_i | grep -e "${version_line_match_i}")
```

Comment:
I think you want to double quote here to avoid globbing. In general for shell scripts I'd run `shellcheck` and address any lint errors it reports.
```suggestion
        version_line=$(cat "$version_file_path_i" | grep -e "$version_line_match_i")
```

Reply:
+1 to [shellcheck](https://github.com/koalaman/shellcheck)